content/docs/pipelines/installation/standalone-deployment.md

```diff
@@ -256,15 +258,9 @@ kubectl delete -k $KFP_MANIFESTS_REPO_LINK
 
 ## Troubleshooting
 
-* If you encounter a permission error when installing Kubeflow Pipelines standalone, run:
-
-```
-kubectl create clusterrolebinding your-binding --clusterrole=cluster-admin --user=<your-user-name>
-```
-
```

Comment:
note, removed this because it only happens in very old Kubernetes clusters that GKE no longer support